File: persephone/provisioning-service/src/main/java/com/vmware/blockchain/deployment/services/orchestration/model/vsphere/NetworkSummary.java
/*
 * Copyright (c) 2020 VMware, Inc. All rights reserved. VMware Confidential
 */

package com.vmware.blockchain.deployment.services.orchestration.model.vsphere;

import lombok.Data;

/**
 * Data class.
 */
@Data
public class NetworkSummary {
    String network;
    String name;
    String type;
}

File: persephone/orchestration-vmc/src/common/com/vmware/blockchain/deployment/model/vsphere/NetworkSummary.kt
/* **************************************************************************
 * Copyright (c) 2019 VMware, Inc.  All rights reserved. VMware Confidential
 * *************************************************************************/
package com.vmware.blockchain.deployment.model.vsphere

import kotlinx.serialization.Serializable

@Serializable
data class NetworkSummary(val network: String, val name: String, val type: String)
